py-rse/teams.Rmd

```diff
@@ -445,17 +465,22 @@ You can't force new issues to be as complete as you might like,
 but you can use an issue template to make it easier for contributors to
 remember and complete documentation about bug reports.
 
-## How can we use labels to organize work? {#teams-labels}
+FIXME: how to respond to a bug report.
```

Comment:
This seems well addressed by the following paragraph. Can the FIXME be removed? Or did you have something else in mind?

Reply:
If you're happy, I'm happy :-)